File: app/src/main/java/com/chartapp/Splash.java
package com.chartapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import com.airbnb.lottie.LottieAnimationView;
import com.chartapp.bottom.Bottom;
import com.chartapp.phones.Phone;


public class Splash extends AppCompatActivity {
    LottieAnimationView anim;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        setTheme(R.style.Theme_AppCompat_DayNight_NoActionBar);
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);



        Thread logoTimer = new Thread()
        {
            public void run()
            {
                try
                {
                    int logoTimer = 0;
                    while(logoTimer < 1500)
                    {
                        sleep(100);
                        logoTimer = logoTimer +100;
                    };
                    startActivity(new Intent(getApplicationContext(), Bottom.class));
                }
                catch (InterruptedException e)
                {

                    e.printStackTrace();
                }
                finally
                {
                    finish();
                }
            }
        };
        logoTimer.start();

    }
}

File: app/src/main/java/com/chartapp/Splash.kt
package com.chartapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.chartapp.R
import android.content.Intent
import com.chartapp.bottom.Bottom

class Splash : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        setTheme(R.style.Theme_AppCompat_DayNight_NoActionBar)
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        val logoTimer: Thread = object : Thread() {
            override fun run() {
                try {
                    var logoTimer = 0
                    while (logoTimer < 1500) {
                        sleep(100)
                        logoTimer = logoTimer + 100
                    }
                    startActivity(Intent(applicationContext, Bottom::class.java))
                } catch (e: InterruptedException) {
                    e.printStackTrace()
                } finally {
                    finish()
                }
            }
        }
        logoTimer.start()
    }
}